bin/fuel-core/src/cli/run/tx_pool.rs

```diff
@@ -0,0 +1,104 @@
+//! Clap configuration related to consensus parameters
+
+use fuel_core::txpool::types::ContractId;
+use fuel_core_types::{
+    fuel_tx::{
+        Address,
+        UtxoId,
+    },
+    fuel_types::Nonce,
+};
+
+#[derive(Debug, Clone, clap::Args)]
+pub struct TxPoolArgs {
+    /// The max time to live of the transaction inside of the `TxPool`.
+    #[clap(long = "tx-pool-ttl", default_value = "5m", env)]
+    pub tx_pool_ttl: humantime::Duration,
+
+    /// The max number of transactions that the `TxPool` can simultaneously store.
+    #[clap(long = "tx-max-number", default_value = "4064", env)]
```

Comment:
Where did the "4064" come from?

Reply:
looks like it was just copied over from before